client/web/src/components/fuzzyFinder/HighlightedLink.tsx

```diff
@@ -36,33 +36,37 @@ export function offsetSum(props: HighlightedLinkProps): number {
 export const HighlightedLink: React.FunctionComponent<HighlightedLinkProps> = props => {
     const spans: JSX.Element[] = []
     let start = 0
-    function pushSpan(className: string, startOffset: number, endOffset: number): void {
+    function pushElement(kind: 'mark' | 'span', startOffset: number, endOffset: number): void {
         if (startOffset >= endOffset) {
             return
         }
         const text = props.text.slice(startOffset, endOffset)
         const key = `${startOffset}-${endOffset}`
-        const span = (
-            <span key={key} className={className}>
-                {text}
-            </span>
-        )
-        spans.push(span)
+        if (kind === 'mark') {
+            spans.push(
+                <mark key={key} className={styles.mark}>
```

Comment:
I just tested this in Chromatic and would suggest removing the padding so that letters don't "jump around" as a bigger part of the path is highlighted when typing.
```suggestion
                <mark key={key} className={styles.mark, 'px-0'}>
```

Reply:
Good idea, done.